test/ext/ws.js

```diff
@@ -88,11 +88,36 @@ describe("web-sockets extension", function () {
         this.messages.length.should.equal(1);
     })
 
-    it('handles message from the server', function () {
```

Comment:
Related to `text` message types, this test was covering that. I tried to add it, but just like the test server, it was failing due to string values having no `text()` method. Please add it back and keep it green, it's primary use case of the extension.

Reply:
I added another test covering `text` message types.